Xamarin.Forms.Core/Button.cs

```diff
@@ -347,6 +348,10 @@ public ButtonContentLayout(ImagePosition position, double spacing)
 
 			public double Spacing { get; }
 
+			public bool IsHorizontal => Position == ImagePosition.Left || Position == ImagePosition.Right;
+
+			public bool IsVertical => Position == ImagePosition.Top || Position == ImagePosition.Bottom;
+
```

Comment:
I'd add `IsNotHorizontal` and `IsNotVertical` too, because they're convenient :P

More seriously, we could do with one of them, but definitely not 2. And the naming should be something else. And if we can't agree, let's drop those inconvenient methods :)

Reply:
Hehe, OK.